t/regulatory.t

```diff
@@ -81,25 +81,26 @@ eq_or_diff($json, $output, 'GET Information about a specific microarray');
 
 my $microarray_probe = '/regulatory/species/homo_sapiens/microarray/HC-G110/probe/137:179;?content-type=application/json';
 $output =  {
-  microarray_name => 'HC-G110',
-  probe_length    => 25,
-  probe_name      => '137:179;',
+  microarray => 'HC-G110',
+  length    => 25,
+  name      => '137:179;',
   sequence        => 'TCTCCTTTGCTGAGGCCTCCAGCTT'
   } ;
 $json = json_GET($microarray_probe,'GET Information about a specific probe');
 eq_or_diff($json, $output, 'GET Information about a specific probe');
 
 
 # regulatory/species/:species/microarray/:microarray/probe_set/:probe_set/probe/:probe
-my $microarray_probe_set_probe = '/regulatory/species/homo_sapiens/microarray/HC-G110/probe_set/1000_at/probe/137:179;?content-type=application/json';
+my $microarray_probe_set = '/regulatory/species/homo_sapiens/microarray/HC-G110/probe_set/1000_at?content-type=application/json';
 $output =  {
- microarray_name => 'HC-G110',
- probe_length    => 25,
- probe_name      => '137:179;',
- probe_set       => '1000_at',
- sequence        => 'TCTCCTTTGCTGAGGCCTCCAGCTT'  
+  microarray => 'HC-G110',
+  name => '1000_at',          
+  probes => [                         
+    '137:179;'                        
+    ],                                  
+  size => 16                          
   } ;
-$json = json_GET($microarray_probe_set_probe,'JSON: GET Information about a specific probe');
+$json = json_GET($microarray_probe_set,'JSON: GET Information about a specific probe');
```

Comment:
Should be "about specific probe set"

Reply:
Yes, good catch! Any chance you update this, or shall I do a new pull request?

Reply:
Once you've created the PR, any new commit you do on your fork will be caught by the PR.